src/footprints/index.js

```diff
@@ -11,5 +11,9 @@ module.exports = {
     rgb: require('./rgb'),
     jstph: require('./jstph'),
     pad: require('./pad'),
-    rotary: require('./rotary')
+    rotary: require('./rotary'),
+	// Fresh prints
+	promicro_front: require('./promicro_front'),
```

Comment:
Like for hotswaps, I think front or back facing should be a parameter on the OG promicro footprint. We have everything we need, so it should be just a little rearrangement.